schema.sql

```diff
@@ -1,4 +1,4 @@
-DROP TABLE IF EXISTS city_explorer;
+DROP TABLE IF EXISTS locations;

```

Comment:
Careful with this line! If you have to make edits to your database with schema.sql, this will delete all your existing data. Nice if you want it to wipe, rough if you want to keep the stuff. Just a heads up!